packages/patternfly-4/react-core/src/components/Alert/__snapshots__/Alert.test.tsx.snap

```diff
@@ -1,25 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Alert - danger Action Close Button 1`] = `
-<Alert
+<Component
```

Comment:
We still need to figure out why these are rendering as <Component>

Reply:
Will have to create a bug against JEST for this it looks like.